### ISSUE TITLE
Parameter `provisionPublicIP` description says "By Default no Public IP addresses are provisioned" yet default setting is "Yes"

### ISSUE THREAD

**Issue by cixelsyd** [F5Networks/f5-aws-cloudformation#88]
## Do you already have an issue opened with F5 support?
Github Issues are consistently monitored by F5 staff, but should be considered as best effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open an case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

NO

## Description
Describe the problem you're having or the enhancement you'd like to request.

The template for `f5-existing-stack-payg-3nic-bigip.template` has the following parameter:

```
 "provisionPublicIP": {
   "AllowedValues": [
    "Yes",
    "No"
   ],
   "Default": "Yes",
   "Description": "Whether or not to provision Public IP Addresses for the BIG-IP Network Interfaces. By Default no Public IP addresses are provisioned.",
   "Type": "String"
  },
```

The parameter description says "by default no public IP addresses are provisioned" but the actual default value for the parameter indicates that public IP will be provisioned.

Whichever way we go - the two should match.

## Template
For bugs, enter the template with which you are experiencing issues below.

`f5-existing-stack-payg-3nic-bigip.template`

## Severity Level
For bugs, enter the bug severity level. Do not set any labels.

Severity: 5

Severity level definitions:
 1. **Severity 1 (Critical)** : Defect is causing systems to be offline and/or nonfunctional. immediate attention is required.
 2. **Severity 2 (High)** : Defect is causing major obstruction of system operations.
 3. **Severity 3 (Medium)** : Defect is causing intermittent errors in system operations.
 4. **Severity 4 (Low)** : Defect is causing infrequent interuptions in system operations.
 5. **Severity 5 (Trival)** : Defect is not causing any interuptions to system operations, but none-the-less is a bug.

**Comment by shyawnkarim** [comment]
Thank you for reporting this issue. I've gone ahead and created a Jira issue, #1740 , to resolve it.

**Comment by shyawnkarim** [comment]
Closing. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-aws-cloudformation-v2).

This issue was resolved with V2.
